docs/source/index.rst

```diff
@@ -6,6 +6,20 @@ Welcome to sktime
 
   A unified framework for machine learning with time series
 
+Google Summer of Code 2022
+==========================
+
+.. admonition:: Join us for Google Summer of Code 2022
+
+   .. figure:: https://summerofcode.withgoogle.com/assets/media/logo.svg
+      :scale: 100 %
+      :alt: GSoC logo
+      :target: events/gsoc_2022.html
+
+
+   :doc:`Check out our GSoC 2022 page for more details </events/gsoc_2022>`
```

Comment:
is it possible to link to the markdown file: https://github.com/sktime/mentoring/blob/main/internships/gsoc2022.md

Otherwise it requires having to update two files for any change.

Reply:
Probably possible but not worth the hassle, I can update the docs manually if you ping me :)